api/MimeType.json

```diff
@@ -29,10 +29,11 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "≤4"
           },
           "safari_ios": {
-            "version_added": true
+            "version_added": "≤3",
+            "version_removed": "8"
```

Comment:
This makes sense. No plugin support on iOS.

Reply:
Hmm, but then the same doesn't apply for `MimeTypeArray` in https://github.com/mdn/browser-compat-data/pull/7864, that's weird. I'm guessing that `navigator.mimeTypes` still exists, but that it's always an empty array? That would make sense to minimize the risk of breaking existing web content. Can you confirm if that's the case in Safari 8 or newer?

Reply:
I tried https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8805 on Safari iOS 10 and indeed `navigator.mimeTypes` exists but is empty. So this seems right.